### ISSUE TITLE
undeclared identifier 'texture2D', 'blur'

### ISSUE THREAD

**Issue by revmischa** [projectM-visualizer/projectm#460]
Built-in idle preset no longer works:
Maybe related to recent changes 

```
2021-02-13 12:32:53.655465+0200 SDLprojectM[6397:95939] INFO: Using data directory: /usr/local/share/projectM
2021-02-13 12:32:53.655678+0200 SDLprojectM[6397:95939] INFO: Config file not found, using built-in settings. Data directory=/Users/cyber/Library/Developer/Xcode/DerivedData/projectM-drbzylqdnfyhklebybsuyullfdrw/Build/Products/Debug/
[PCM] MAX SAMPLES:2048
Failed to compile shader 'Fragment: v2f_c4f_t2f'. Error: ERROR: 0:13: Invalid call of undeclared identifier 'texture2D'

Failed to compile shader 'Fragment: blur1'. Error: ERROR: 0:37: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:38: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:40: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:41: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:43: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:44: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:46: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:47: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:50: Use of undeclared identifier 'blur'
ERROR: 0:52: Use of undeclared identifier 'blur'
ERROR: 0:52: Use of undeclared identifier 'blur'
ERROR: 0:54: Use of undeclared identifier 'blur'

Failed to compile shader 'Fragment: blur2'. Error: ERROR: 0:34: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:35: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:37: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:38: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:40: Use of undeclared identifier 'blur'
ERROR: 0:47: Use of undeclared identifier 'blur'
ERROR: 0:49: Use of undeclared identifier 'blur'
```

**Comment by revmischa** [comment]
on macOS Big Sur

**Comment by revmischa** [comment]
Not sure of the cause; maybe related to 093164d583e758794e93b8d965722095d0a04484

**Comment by revmischa** [comment]
Fixed I think as part of #461
